src/core/localization/gettext/ar/common.po

```diff
@@ -75,131 +73,6 @@ msgstr "عرض العنوان التفسيري"
 msgid "Layers"
 msgstr "طبقات"
 
-#: bivariate##color_manager##not_defined
-msgid "Not defined"
-msgstr "غير معرف"
-
-#: bivariate##color_manager##sentiments_combinations_filter
-msgid "Sentiments combinations"
-msgstr "تركيبات المشاعر"
-
-#: bivariate##color_manager##no_legends
-msgid "There are no legends satisfying the conditions."
-msgstr "لا توجد ملاحظات تفي بالشروط."
-
-#: bivariate##color_manager##no_data
-msgid "No Data."
-msgstr "لايوجد بيانات."
-
-#: bivariate##color_manager##sentiments_tab
-msgid "Sentiments"
-msgstr "المشاعر"
-
-#: bivariate##color_manager##color_legends_tab
-msgid "Color Legends"
-msgstr "مؤشرات اللون"
-
-#: bivariate##color_manager##layers_tab
-msgid "Layers (indicators)"
-msgstr "الطبقات (المؤشرات)"
-
-#: bivariate##panel##header
-msgid "Bivariate Matrix"
-msgstr "مصفوفة ثنائية المتغير"
-
-#: bivariate##matrix##caption##base_axis
-msgid "Base axis"
-msgstr "المحور الأساسي"
-
-#: bivariate##matrix##caption##annex_axis
-msgid "Annex axis"
-msgstr "المحور الملحق"
-
-#: bivariate##matrix##caption##tooltip##p1
-msgid ""
-"Charts involving the Base and Annex axes help establish a relationship "
-"between two data sets."
-msgstr ""
-"تساعد المخططات التي تتضمن محوري القاعدة والملحق في إنشاء علاقة بين مجموعتي "
-"بيانات."
-
-#: bivariate##matrix##caption##tooltip##li1
-msgid "Annex axis - parameters that we analyse"
```

Comment:
why is this shuffled around the file? it looks like we did not pass the files through msgmerge with our original .po file

Reply:
I found that translator applied some formatting - str length + keys order is changed.
But all keys are here.

Reply:
And yeah, I didn't run msgmerge, should we? Just not familiar with this tool.

Reply:
we don't this step implemented :/
<img width="744" alt="Screenshot 2022-12-21 at 14 56 41" src="https://user-images.githubusercontent.com/20676525/208899883-196620aa-774d-4e9a-871c-a5e49214f108.png">

Reply:
ah, it was just incorrectly placed first time. 
now first bivaraite key #: bivariate##panel##header goes right after #: drawings##self_directions_not_supported
like in our POT, so it's all correct